ClosedXML/Excel/XLWorkbook_Load.cs

```diff
@@ -933,7 +933,7 @@ private static bool IsRowReference(string sheetArea)
         private static void ParseReference(string item, out List<Tuple<string, string>> sheetNameAndArea) 
         {
             sheetNameAndArea = new List<Tuple<string, string>>();
-            ParseTreeNode nodes = ExcelFormulaParser.Parse("Test(" + item + ")");
+            ParseTreeNode nodes = ExcelFormulaParser.Parse(String.Format("DummyFormula({0})", item));

```

Comment:
For the library to parse the print area, it had to be mocked up in a formula format.  Once parsed, the information is dropped.